src/main/java/uk/gov/justice/hmpps/prison/service/PrisonerSearchService.kt

```diff
@@ -46,7 +47,8 @@ class PrisonerSearchService(
           csra = it.csra,
           categoryCode = it.categoryCode,
           inOutStatus = it.inOutStatus,
-          identifiers = it.identifiers?.sortedBy { it.whenCreated },
+          identifiers = it.identifiers?.filter { id -> id.offenderId == it.offenderId }?.sortedBy { it.whenCreated },
+          allIdentifiers = it.identifiers?.sortedBy { it.whenCreated },
```

Comment:
But `identifiers` will be replaced by `allIdentifiers` so we can see identifires for aliases.